resource/js/make-plugin-callbacks.js

```diff
@@ -0,0 +1,29 @@
+const makeCallbacks = () => {
+  const newPage = window.SKOSMOS.pageType
+  const newUri = window.SKOSMOS.uri
+  const newPrefs = window.SKOSMOS.prefLabels
+  const ldJsonScript = document.querySelector('script[type="application/ld+json"]')
```

Comment:
This works for now, but I think it would be better to give the script element containing the JSON-LD an `id` attribute and use `getElementById` here.